src/components/molecules/BenefitFilter.js

```diff
@@ -18,7 +18,7 @@ export function BenefitFilter(props) {
           text={props.eligibleText}
           invert={props.isSelectedEligible === false}
           onClick={handleClick}
-          className={"ml-0"}
+          className={"ml-0 bg-gray-700 text-white hover:bg-black"}
```

Comment:
Should the classes be consistent here? - Either with `dk` or `700` everywhere (also lines 30, 39) you changed it to `dk` in the Action Button.
```suggestion
          className={"ml-0 bg-gray-dk text-white hover:bg-black"}
```

Reply:
Hello Shaun, thanks for the review! You are absolutely right, I forgot to use the customized css name for this component, thanks for point it out.